doc/classes/VoxelTool.xml

```diff
@@ -41,39 +41,47 @@
 			<param index="0" name="begin" type="Vector3i" />
 			<param index="1" name="end" type="Vector3i" />
 			<description>
-				Operate on a rectangular cuboid section of the terrain. [code]begin[/code] and [code]end[/code] are inclusive. Choose operation and which voxel to use by setting [code]value[/code] and [code]mode[/code] before calling this function.
+				Operate on a rectangular cuboid section of the terrain. [code]begin[/code] and [code]end[/code] are inclusive. 
+				You must choose operation by setting [code]mode[/code] and set parameters relevant to the type of tool before calling this function.
```

Comment:
At first I thought of linking every do_* op to Box as the most documented one, but now I'm thinking that maybe this and the next comment can be moved to ''[Using VoxelTool](https://voxel-tools.readthedocs.io/en/latest/scripting/#using-voxeltool)" section of Scripting article in the main doc. Maybe even with code examples and stuff. Does it sound reasonable?

Reply:
It would be nice, but at the same time it requires putting absolute links in there, while it could also be in the description of the class. But I dont think we can link to that description somehow.